.travis.yml

```diff
@@ -5,5 +5,5 @@ before_install:
 - gem install xcpretty --no-document
 script:
 - set -o pipefail
+- git submodule update --init --recursive
 - xcodebuild -workspace JWT.xcworkspace -scheme JWT test -sdk macosx | xcpretty -c
-- pod lib lint
```

Comment:
Can you restore this line, it's verifying if the pod spec is valid for CocoaPods.